### PR TITLE
Full-Text Search Blogpost

### PR DESCRIPTION
**Pull request by ZoeLeBlanc**
Blog post announcing full-text search, with details on how to use the search feature, how it works, and how we built the functionality.

### Checklist

- [X] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
  ~- [ ] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description~
- [x] Add the appropriate "Label"
- [x] [Ensure the Travis CI checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [ ] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing Travis errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Travis Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-travis-errors). Then contact the technical team if you need further help.*

### CODE REVIEW

**Comment by ZoeLeBlanc** [comment]
@programminghistorian/technical-team can someone help me figure out this build error? Seems to be a link error from the html checker. When I serve and build locally I have not problems so not sure what to do next, beyond just pushing again 😅 😬  

Thanks 🙏

**Comment by mdlincoln** [comment]
This means an old link has broken - our policy is you find it, you fix it, usually by replacing the link with one from http://web.archive.org/

File: _posts/2020-05-26-full-text-search.md
Comment (mdlincoln) [inline _posts/2020-05-26-full-text-search.md]: Check this and other links that appear later in the post - you haven't got the markdown linking quite right - it's `[text here](url)`

**Review by mdlincoln — CHANGES_REQUESTED** [review]
I love the explanation! Just check the formatting of your links.

This is also a good chance to update your bio in ph-authors.yml - it's out of date 😬

File: _posts/2020-05-26-full-text-search.md
Comment (ZoeLeBlanc) [reply _posts/2020-05-26-full-text-search.md]: haha oh jeez 😅 ! Thanks for catching that.... I swear I've used markdown before 🙈

File: _posts/2020-05-26-full-text-search.md
Comment (mdlincoln) [inline _posts/2020-05-26-full-text-search.md]: I more expected you to do something like:

```md
information retrieval algorithm called [Okapi BM25](https://en.wikipedia.org/wiki/Okapi_BM25)
```

for most of these links, so we don't have long urls hanging all over the place?

Comment (ZoeLeBlanc) [reply _posts/2020-05-26-full-text-search.md]: very reasonable request that I just implemented 👍

**Review by mdlincoln — DISMISSED** [review]
Sweet! merge when ready

**Comment by ZoeLeBlanc** [comment]
Just added a few small tweaks to the search code after writing the docs. @mdlincoln do you mind giving me approval again once checks pass? I'll merge in once I have the all clear!

**Review by mdlincoln — APPROVED** [review]

**Pull request merged**
